gowin/cells.cc

```diff
@@ -83,14 +84,14 @@ void lut_to_lc(const Context *ctx, CellInfo *lut, CellInfo *lc, bool no_dff)
     }
 
     if (no_dff) {
-        lc->params[id_FF_USED] = 0;
+        lc->ff_used = false;
```

Comment:
Sorry, I may have explained this badly before. The correct way to do it is to set a param here and then set the ArchCellInfo field to mirror that param in `assignArchInfo` - ArchCellInfo should never be the only storage for important things, to avoid creating any more problems for loading/storing designs or post/pack simulations than needed (even if both these things are admittedly experimental right now)

Reply:
FF_USED is not actually used on the Apicula side of things, so it's no important info. Actually the only place it's used is in validation that no DFF are used in the top two slices that don't have any.

Reply:
And that would break if you saved the netlist between pack and place